plugins/aws-s3-storage/src/index.ts

```diff
@@ -227,6 +195,68 @@ export default class S3Database implements IPluginStorage<S3Config> {
     return this._localData as LocalStorage;
   }
 
+  private async _getPackageList(): Promise<any> {
+    const logger = this.logger;
+    const { bucket, keyPrefix } = this.config;
+    const listPath = keyPrefix.endsWith('/') ? keyPrefix : '/';
```

Comment:
This shouldn't need editing. See `addTrailingSlash` helper added in #313 (https://github.com/verdaccio/monorepo/pull/313/files#diff-362e190e13ca3d0e1e54d19c7a3f45e0R38)

Reply:
Looks good